src/model.js

```diff
@@ -422,13 +432,22 @@ const BookshelfModel = ModelBase.extend({
   morphTo(morphName) {
     if (!_.isString(morphName)) throw new Error('The `morphTo` name must be specified.');
     let columnNames, candidates;
-    if (_.isArray(arguments[1])) {
-      columnNames = arguments[1];
+    if (_.isArray(arguments[1]) || _.isNil(arguments[1])) {
+      columnNames = arguments[1] || null;
```

Comment:
This would mean both:

```
let Photo = bookshelf.Model.extend({
  tableName: 'photos',
  imageable: function() {
    return this.morphTo('imageable', ["ImageableType", "ImageableId"], [Post, "cover"], Site);
  }
});
```

and

```
let Photo = bookshelf.Model.extend({
  tableName: 'photos',
  imageable: function() {
    return this.morphTo('imageable', [Post, "cover"], Site);
  }
});
```

Would be valid?